acvm_backend_barretenberg/Cargo.toml

```diff
@@ -22,8 +29,16 @@ blake2 = "0.9.1"
 tempfile = "*"
 
 [features]
-default = [
-    # "acvm/bn254",
+default = ["wasm"]
```

Comment:
I'm playing around with this locally and I'm seeing build failures when I turn this to `"native"` so I think this needs some more work.

Reply:
Oops, looks like this snuck in when merging #133 back in. Fixed.